camera-orderfile/README.md

```diff
@@ -0,0 +1,95 @@
+# NdkCamera Sample
+
+Two API samples:
+
+- texture-view: Preview NDK camera image with
+  [Android TextureView](https://developer.android.com/reference/android/view/TextureView.html)
+- basic: A basic NdkCamera sample to preview camera images with AReadImage and
+  take jpeg photos. Exposure and sensitivity are adjustable for preview, however
+  capturing photos is in auto mode (it could be adjustable with similar method
+  as used for preview).
+
+## Other Resources
+
+- Getting familiar with the 5 Camera2 objects
+  ![Camera2 API Model](Camera2ProgrammingModel.png)
+
+- [Camera2 blogs](https://medium.com/androiddevelopers/camera-enumeration-on-android-9a053b910cb5)
+
+- [Camera2 Java documentation](https://developer.android.com/reference/android/hardware/camera2/package-summary)
+
+## Pre-requisites
+
+- Android Studio 2.3.0+ with [NDK-r15+](https://developer.android.com/ndk/)
+  bundle
+- Android device running android-24+
+
+## Getting Started
+
+1. Download Android Studio from
+   [latest stable release](http://developer.android.com/sdk/index.html) or
+   [canary](http://tools.android.com/download/studio/canary)
+1. Launch Android Studio
+1. Select "Import project (Eclipse ADT, Gradle,etc)"
+1. Browse into downloaded sample directory, select webp/build.gradle
+1. Click *Tools/Android/Sync Project with Gradle Files*.
+1. Click *Run/Run 'app'*.
+
+## Screenshots
+
+![screenshot](ndkCamera.png)
+
+## Order file Functionality
+
+Compared to the normal camera sample, this one has order file profiling and loading added to it. We have a basic order file example in `orderfile`. The `orderfile` example is in Kotlin while this sample is in pure Java so there are some differences. One main difference is the order files are dumped when the app is paused rather than at the startup. Here are the steps for order file in this sample: 
```

Comment:
These differences probably aren't actually necessary? I expect it's fine to reduce to one sample and just explain the alternate spelling for the other language in a comment.